fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/observer/eventmappings/AllNodeEventsOneEventTest.java

```diff
@@ -78,14 +78,11 @@
     @Mock
     private Event mockEvent5;
 
-    @Mock
-    private Iterator<Event> mockIterator;
+    private Stream<Event> mockStream;
```

Comment:
:+1:  to using actual `Stream`s for this testing. Maybe use actual `Event`s, too.